test.js

```diff
@@ -1,93 +1,301 @@
-var assert = require("assert");
-var generate = require("bit-docs-generate-html/generate");
-var path = require("path");
-
-var Browser = require("zombie"),
-	connect = require("connect");
-
-
-var find = function(browser, property, callback, done){
-	var start = new Date();
-	var check = function(){
-		if(browser.window && browser.window[property]) {
-			callback(browser.window[property]);
-		} else if(new Date() - start < 2000){
-			setTimeout(check, 20);
-		} else {
-			done("failed to find "+property);
+var path = require('path'),
+	rmrf = require('rimraf'),
+	assert = require('assert'),
+	Browser = require('zombie'),
+	express = require('express'),
+	generate = require('bit-docs-generate-html/generate');
```

Comment:
For me, the odd spacing would stand out as a bigger issue when it comes to "easier to scan through the list if they were formatted the same way", and assumed that's what you meant. 

The fact that I'm using tabs instead of spaces is a testament to the fact that I'm not having personal preference about formatting. I will change it however you like, but I was mostly concerned about getting your feedback on the functionality and test coverage!